php/exceptions/NotFound.class.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
```

Comment:
Note: This (an EmptyStream) were previously used by changes to the Module and NDB_Page handler in this PR, which I had to rollback because they were incompatible with the existing implementation, putting a lie to the claim that this PR doesn't change any behaviour yet.

They'll be used in the next PR, when the whole thing gets switched over and things start getting used..